packages/metadata/src/nftItem/index.ts

```diff
@@ -39,5 +39,15 @@ export type NftItem = Omit<IItemMetadata, "type"> & {
   };
   quantity?: number;
   transferMethod?: string;
-  transferDelay?: string;
+  terms?:
+    | {
+        key: string;
+        value: string;
+        displayType?: string;
+      }[]
+    | {
+        key: string;
+        value: string;
+        display_type?: string;
+      }[];
```

Comment:
No need to accept non camel case here (the only reason why it's allowed for `attributes` is because Opensea defines it with underscore)

```suggestion
  terms?: {
    key: string;
    value: string;
    displayType?: string;
  }[];
```